examples/mongodbatlas_privatelink_endpoint/aws/cluster-geosharded/README.md

```diff
@@ -86,7 +86,7 @@ $ terraform destroy
 2. `mongodbatlas_privatelink_endpoint` is dependent on the `mongodbatlas_project`
 3. `aws_vpc_endpoint` is dependent on the `mongodbatlas_privatelink_endpoint`, and its dependencies.
 4. `mongodbatlas_privatelink_endpoint_service` is dependent on `aws_vpc_endpoint` and its dependencies.
-5. `mongodbatlas_cluster` is dependent only on the `mongodbatlas_project`, howerver; its `connection_strings` are sourced from the `mongodbatlas_privatelink_endpoint_service`. `mongodbatlas_privatelink_endpoint_service` has explicitly been added to the `mongodbatlas_cluster` `depends_on` to ensure the private connection strings are correct following `terraform apply`.
+5. `mongodbatlas_advanced_cluster` is dependent only on the `mongodbatlas_project`, howerver; its `connection_strings` are sourced from the `mongodbatlas_privatelink_endpoint_service`. `mongodbatlas_privatelink_endpoint_service` has explicitly been added to the `mongodbatlas_advanced_cluster` `depends_on` to ensure the private connection strings are correct following `terraform apply`.
```

Comment:
The end of this: "strings are correct following "terraform apply" " --  doesn't read well and could use an edit. 

General recommendation: Is there a way to write these in active voice? Are we asking users to do this, so that the dependencies are all satisfied? Or is this done by code?

The way this is written in passive voice will make it difficult for users to know that they are asked to do something, if that's the case. 

All of these bullets could be rewritten in active voice and start with imperative verbs, such as: Add `abc` to `def`. This ensures that ...

Reply:
thanks, i've changed the end you say and voices to be active, done in: https://github.com/mongodb/terraform-provider-mongodbatlas/pull/2453/commits/72387e7911dc961b7d12e11ceb5cbfe04d84bfc6